addons/core/translations/resources/en-us.yaml

```diff
@@ -708,17 +706,19 @@ target:
       dual-egress-on: In this configuration, the client will attempt to  connect through any client-accessible "frontline" worker and reach the host through any filtered egress worker.
       dual-egress-on-ingress-on: In this configuration, the client will attempt to connect through any filtered ingress worker and reach the host through any filtered egress worker.
       hcp-dual-egress-on: In this configuration, the client will attempt to connect through any HCP-managed worker and reach the host through any filtered egress worker.
-    filter:
-      title: Filter
-      description: Boolean expression.
-    egress_worker_filter:
-      title: Egress worker filter
+    egress-workers:
+      title: Egress workers
       description: Specify workers that have access to the target (such as within a private network).
-    ingress_worker_filter:
-      title: Ingress worker filter
+    ingress-workers:
+      title: Ingress workers
       description: Specify workers Boundary should use to start the route to this target.
+    worker-filter:
+      title: Worker filter
+      description: The worker filter format for this target.
     actions:
-      update-filter: Update Filter
+      add-ingress-worker-filter: Add ingress worker filter
+      add-egress-worker-filter: Add egress worker filter
```

Comment:
Should these be uncapitalized as the other actions seems like they are

Reply:
hmm the other actions in the "Manage" dropdown are capitalized

Reply:
ah nvm i see it now, yeah ill update this.